### PR TITLE
Slovak translation update

### PR DESCRIPTION
**Pull request by Roxterat**

### CODE REVIEW

**Comment by Roxterat** [comment]
Itsy bitsy Roxterat calling itsy bitsy @zenseii for a review..

File: files/lang/sk.po [hunk 1]
```diff
@@ -677,287 +677,288 @@ msgid "View Hero's options"
...
-msgstr ""
+msgstr "%{name} strieľa so silou %{count} lukostrelcov"
 
 msgid "each with a +%{attack} bonus to their attack skill."
-msgstr ""
+msgstr "s bonusom +%{attack} k ich zručnosti útoku."
 
 msgid "The %{name} is destroyed."
-msgstr ""
+msgstr "%{name} je zničený/á."
```
[the remark this answers is not in the document]
Comment (Roxterat) [reply files/lang/sk.po:886]: Unfortunately for Towers it's "zničená veža" (female) and for Ballista it's "zničený balista" (male), but I'll try to think of a genderless alternative..

**Comment by Roxterat** [comment]
> Hi, @Roxterat. I haven't had the chance to check the actual translations, but I left one general comment.
> 
> Please have a look when you have the time.

Replied, but it's not resolved yet, I will tinker with it a bit more..

**Comment by Roxterat** [comment]
> Hi, @Roxterat.
> 
> Were you able to find something better? If not should we just proceed with the review of this PR? The Italian translation has similar use of forward slashes so it's not prohibited, just preferrably avoided.

I was trying to think of something but didn't figure so far. I will return back to this for the sake of the player's experience, either later or as soon as something acceptable comes to my mind.
Other than that this PR would be ready for a review.

File: files/lang/sk.po [hunk 2]
```diff
@@ -677,287 +677,288 @@ msgid "View Hero's options"
...
-msgstr ""
+msgstr "Corlagonova obrana"
 
 msgid "The Crown"
-msgstr ""
+msgstr "Koruna"
 
 msgid "The Gauntlet"
-msgstr ""
+msgstr "Rukavica"
```
[the remark this answers is not in the document]
Comment (Roxterat) [reply files/lang/sk.po]: > It seems like this is translated as Gauntlet, similar to a glove. This string refers to the definition of Gauntlet as a race or a trial: https://en.wikipedia.org/wiki/Running_the_gauntlet
> 
> This map is about having limited time to reach the oppononent and defeating them.

Well spotted, I didn't knew it refers to this sort. Changed to "Výzva", meaning "Challenge" and actually "Gauntlet" in this concept as well.

**Comment by Roxterat** [comment]
> Hi, @Roxterat. I left one single comment. Please have a look when you have the time. If my assumption is wrong then you can simply ignore it.

Hey @zenseii , I've replaced the Gauntlet thing properly, didn't knew it was that case. Also I hope marking conversations as "resolved" is okay once we discussed the topic at that point.

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@Roxterat , huge thanks for this improvement!

**Pull request merged**